### DISCUSSION TITLE
Filter the plugin list based the configuration items they contain

### DISCUSSION THREAD

**Discussion by JuusoKo** [runelite/runelite#16594] [Ideas]
Often times I don't necessarily remember the name of a plugin that's displaying information on the screen, for example: 'Dude, where's my stuff' was showing an active deathbank, which I did not want to be shown.

What a configuration option -based filtering would allow is to filter the list of plugins based on if they contain the string 'deathbank' or not. In this example, the option was called 'show deathbank infobox'

**Comment by Felanbird** [comment]
The ability to search plugins based on tags already exists, the dude-wheres-my-stuff plugin just doesn't have any set.

You should instead [open an issue](https://github.com/Thource/dude-wheres-my-stuff/issues) asking them to add some search tags, you can let them know it should be added as apart of their `PluginDescriptor` found [here](https://github.com/Thource/dude-wheres-my-stuff/blob/main/src/main/java/dev/thource/runelite/dudewheresmystuff/DudeWheresMyStuffPlugin.java#L64).

**Comment by raiyni** [comment]
You should be able to hold shift and right click an overlay to see which plugin it comes from.
